index.html

```diff
@@ -14193,15 +14176,17 @@ <h4 id="PrimaryGlobal">4.3.6. [Global] and [PrimaryGlobal]</h4>
                   </ol>
                 </li>
                 <li>Truncate <var>values</var> to have length <var>count</var>.</li>
-                <li>Let <var>script</var> be the <a class="dfnref" href="#dfn-callback-context">callback context</a> associated with <var>V</var>.</li>
-                <li>Push <var>script</var> on to the <a class="dfnref external" href="http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html">stack of incumbent scripts</a>. <a href="#ref-HTML">[HTML]</a></li>
+                <li>Let <var>execution context</var> be <a class="external" href="https://tc39.github.io/ecma262/#sec-getfunctionrealm">GetFunctionRealm</a>(<var>F</var>)'s <a class="dfnref external" href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-settings-object">settings object</a>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#realm-execution-context">realm execution context</a>.</li>
+                <li>Increment <var>execution context</var>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#entrance-counter">entrance counter</a> by one.</li>
```

Comment:
Incumbent is meant to be in a follow-up PR. But it sounds like from the rest of the comments there's still a lot of work to do to get entry settings right too... will try to work on it :-/